swagger.json

```diff
@@ -305,9 +305,9 @@
           "description": "File Path",
           "type": "string"
         },
-        "qcstatus": {
-          "description": "FileQC status (PASS, FAIL, PENDING)",
-          "type": "string"
+        "qcpassed": {
+          "description": "Is FileQC PASS? (null = PENDING)",
```

Comment:
why isn't PENDING also false?

Reply:
There are three states: PASS (true), FAIL (false), PENDING (no value). Although now that I'm thinking it through, no FileQCs will have PENDING value (only PASS/FAIL) so I've removed this part of the comment.

Reply:
(PENDING only shows up in the endpoints that request data from both FileQC and FPR, to indicate matching FPR records that have no FileQC value. This endpoint is FileQCs only, and FileQCs will only ever have qcpassed = true/false)